src/README.md

```diff
@@ -0,0 +1,16 @@
+## Usage
+
+### Prerequisites
+
+```bash
+pip install -e .
+cd..
```

Comment:
Why would you need to go up a level?
```suggestion
```

Reply:
Otherwise the command pip install -e . does not work

Reply:
But therefore the command would need to be executed before the pip install -e ., or?
Also could we please shift this Readme.md to the upper layer instead of inside of the src directory ?